src/hevm/src/EVM/Types.hs

```diff
@@ -244,21 +236,18 @@ data Expr (a :: EType) where
 
   -- logs
 
-  EmptyLog       :: Expr Logs
-
-  Log            :: Expr EWord         -- address
+  LogEntry       :: Expr EWord         -- address
```

Comment:
I made a small change to the way that we represent logs here, so we just use a normal haskell list of `LogEntry`s, which made working with the logs in `Format.hs` much easier.